gavsearch.java

```diff
@@ -9,20 +9,15 @@
 import de.codeshelf.consoleui.prompt.ConsolePrompt;
 import de.codeshelf.consoleui.prompt.ListResult;
 import de.codeshelf.consoleui.prompt.PromtResultItemIF;
-import jline.console.completer.StringsCompleter;
 import org.fusesource.jansi.AnsiConsole;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Parameters;
 
-import javax.naming.directory.SearchResult;
 import javax.ws.rs.client.ClientBuilder;
-import javax.ws.rs.core.Response;
-import java.awt.*;
```

Comment:
without this line it can't actually import as "Toolkit" can't be resolved.

Reply:
Hmm... My IDE removed that for me somehow. Let me try to fix that.